docs/guide.md

```
@@ -468,6 +468,29 @@ This will display:
+```sh
+./mump2p tracer dashboard
+```
+
+**Options:**
+- `--window`: Time window for metrics (default: `10s`)
+- `--topic`: Topic for auto-publishing demo messages (default: `demo`)
+- `--count`: Number of messages to auto-publish (default: `60`)
+- `--interval-ms`: Interval between published messages in ms (default: `500`)
+
+**Example:**
+```sh
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)